libraries/lib-wave-track/WaveTrack.h

```diff
@@ -398,8 +399,36 @@ class WAVE_TRACK_API WaveTrack final
    //
    Envelope* GetEnvelopeAtTime(double time);
 
+   const WaveClip* GetClipAtTime(double time) const;
    WaveClip* GetClipAtTime(double time);
 
+   /*!
+    * @brief Returns clips next to `clip` in the given direction, or `nullptr`
+    * if there is none.
+    */
+   const WaveClip* GetNextClip(
+      const WaveClip& clip, PlaybackDirection searchDirection) const;
+   /*!
+    * @copydoc GetNextClip(const WaveClip&, PlaybackDirection) const
+    */
+   WaveClip*
+   GetNextClip(const WaveClip& clip, PlaybackDirection searchDirection);
+
+   /*!
+    * @brief Similar to GetNextClip, but returns `nullptr` if the neighbour
+    * clip is not adjacent.
+    */
+   const WaveClip* GetAdjacentClip(
+      const WaveClip& clip, PlaybackDirection searchDirection) const;
+   /*!
+    * @copydoc GetAdjacentClip(const WaveClip&, PlaybackDirection) const
+    */
+   WaveClip*
+   GetAdjacentClip(const WaveClip& clip, PlaybackDirection searchDirection);
+
+   WaveClipConstHolders GetOverlappingClips(double t0, double t1) const;
+   WaveClipHolders GetOverlappingClips(double t0, double t1);
```

Comment:
Declared but not defined in any of the six PRs.

Reply:
Ok, then it's not used. I'll remove. Thanks.